.mergify.yml

```
@@ -8,3 +8,11 @@ pull_request_rules:
       merge:
         method: squash
         strict: true
+  - name: backport patches to v0.41.x branch
+    conditions:
+      - base=master
+      - label=backport-to-v0.41.x
```

```suggestion
      - label=backport/0.41.x (Stargate)
```

i guess we could just use the existing one, it's used exactly for this purpose (PRs on master that need to be backported)

---

Can we adopt a different terminology, please? I'm proposing to replace `backport` with `cherry-pick`. It's technically more correct (and it's mergify's default terminology too)

---

oh okay, I see people marking the actual backport PR with that label so didnt think it was for that. Good to know we can use it on the original PR. 



> Can we adopt a different terminology, please? I'm proposing to replace `backport` with `cherry-pick`. It's technically more correct (and it's mergify's default terminology too)

The mergify term is backport. (https://mergify.io/features/backports) Dont have a preference. Feel free to push to this pr with what has consensus

---

Meh, ok -.- Let's stick to it